src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsTable.tag

```diff
@@ -0,0 +1,79 @@
+<%@ tag description="instructorFeedbacks - feedback sessions table/list" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ tag import="teammates.common.util.FieldValidator" %>
+<%@ tag import="teammates.logic.core.Emails.EmailType" %>
+
+<%@ attribute name="fsList" type="teammates.ui.template.FeedbackSessionsTable" required="true"%>
+
+<table class="table-responsive table table-striped table-bordered" id="table-sessions">
+    <thead>
+        <tr class="fill-primary">
+            <th id="button_sortid" onclick="toggleSort(this,1);"
+                class="button-sort-ascending">Course ID <span
+                class="icon-sort unsorted"></span>
+            </th>
+            <th id="button_sortname" onclick="toggleSort(this,2)"
+                class="button-sort-none">
+                    Session Name <span class="icon-sort unsorted"></span>
+            </th>
+            <th>Status</th>
+            <th>
+                <span title="<%= Const.Tooltips.FEEDBACK_SESSION_RESPONSE_RATE %>"
+                    data-toggle="tooltip" data-placement="top">
+                    Response Rate
+                </span>
+            </th>
+            <th class="no-print">Action(s)</th>
+        </tr>
+    </thead>
+        <c:choose>
+            <c:when test="${not empty fsList.existingFeedbackSessions}">
+                <c:forEach items="${fsList.existingFeedbackSessions}" var="sessionRow" varStatus="i">
+                     <tr id="session${i.index}"
+                     
+                        <c:forEach items="${sessionRow.rowAttributes.attributes}" var="attr"
+                        > ${attr.key}="${attr.value}"</c:forEach>  >
+                     
+                     
+                        <td>
+                            ${sessionRow.courseId}
+                        </td>
+                        <td>
+                            ${sessionRow.name}
+                        </td>
```

Comment:
I think we can clean it up a little here by removing unnecessary blank lines or using single lines for shorter HTML elements instead of splitting them into multiple lines.